### PR TITLE
Fix memory leaks in statistics handling

### PR DESCRIPTION
**Pull request by MarkRiddoch**
FOGL-3694: adds EDS compatibility

### CODE REVIEW

**Review by pintomax — DISMISSED** [review]

File: C/services/south/ingest.cpp [hunk 1]
```diff
@@ -86,6 +86,7 @@ int Ingest::createStatsDbEntry(const string& assetName)
 				return -1;
```
Comment (AmandeepArora) [inline C/services/south/ingest.cpp:87]: I think result should be freed here also.

**Review by AmandeepArora — DISMISSED** [review]
One minor comment, looks good otherwise.

File: C/services/south/ingest.cpp (hunk 1 above)
Comment (MarkRiddoch) [reply C/services/south/ingest.cpp:87]: Yes, thanks for picking that additional one up

**Review by AmandeepArora — APPROVED** [review]

**Review by pintomax — APPROVED** [review]

**Comment by ashish-jabble** [comment]
@MarkRiddoch Anything pending on the pull request? fyi, I ran a custom CI JOB with the branch on platforms (ub1804, RPI with buster) and I couldn't find any regression with it. I think we can merge this PR under foglamp organization.

**Pull request merged**